### DISCUSSION TITLE
When will be the release date for AdoPiSoft V6?

### DISCUSSION THREAD

**Discussion by VinSheen** [AdoPiSoft/Beta-Releases#38] [General]
When will be the release date for AdoPiSoft V6?
Ado V5.X.X series seems to be having a lot issues as compared to V4.3.0 before.

Maybe we SHOULD consider the system's stability and sustainability first before jumping into a lot of features and updates.

We, vendo builders  and vendo owners, would still prefer stability and usability rather than adding a lot of features which has a lot of issues too.
![image](https://user-images.githubusercontent.com/91281244/163024705-93315260-e1aa-4c7e-863d-adf685e393ba.png)

**Comment by hasap86** [comment]
V6 indonesian language for admin and captive portal :)
